src/components/05_pages/NodeForm/UiMetadata.js

```diff
@@ -44,6 +44,13 @@ const UiMetadata = {
       suffix: ' minutes',
     },
   },
+  field_tags: {
```

Comment:
We should at least open a follow-up to support auto creating entities. The recipes content type doesn't use that on the tags field but the articles content type does.